package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "eccube-2_13",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "git@github.com:nobuhiko/eccube-2_13.git",
```

```suggestion
  "repository": "git@github.com:EC-CUBE/eccube-2_13.git",
```
`EC-CUBE/eccube-2_13` にしておいた方が良いかも

---

直しました